GTBitsOfGood/enchanted-closet
# #212: Initial Implementation of the Reset Password UI

# Overview

Created the following elements for the frontend:
- Component: ResetPasswordForm
- Container: ResetPassword
- Route: /reset-password

Modified the following pre-existing frontend elements:
- Container: Login

Note the Reset Password UI does not support the following features yet:
- Display a loading message
- Dispatch any redux actions

# Testing Done
- Manual testing: Ensure UI looks good
- Manual testing: Ensure routes work
- Manual testing: Ensure form validation works

# UI Preview

**Reset Password Container**

This is new UI.

![image](https://user-images.githubusercontent.com/11142171/48170511-ded90780-e2c5-11e8-9db9-a16c119f164e.png)

**Login Container**

Note I had to add a button under the Login Form.

![image](https://user-images.githubusercontent.com/11142171/48170551-0c25b580-e2c6-11e8-920b-8632a5b20d05.png)
